rust/libnewsboat/src/utils.rs

```diff
@@ -620,6 +620,23 @@ pub fn strip_comments(line: &str) -> &str {
     &line[0..first_pound_chr_idx]
 }
 
+/// Extract filter and url from line separated by ':'.
+pub fn extract_filter(line: &str) -> (&str, &str) {
+    debug_assert!(line.starts_with("filter:"));
```

Comment:
`extract_filter` shouldn't panic even if handed invalid inputs—we even have a test for thta. Let's not assert anything in here.

Reply:
This means you'll have to handle the errors, below, instead of calling `unwrap` and `expect`.

Reply:
From what I see right now, error handling is not using `Result` right?

Reply:
That line is just to confirm that it must start with `filter:`, right now from the code, it is already confirmed to start with filter on the other parts in C++ code.

Reply:
Yep, you're right! Sorry about that. Let's keep `debug_assert` then.